science_rcn/learning.py

```diff
@@ -183,8 +183,8 @@ def adjust_edge_perturb_radii(frcs,
         round_down_error = total_rounding_error + lower - desired_radius
         if abs(round_up_error) < abs(round_down_error):
             graph.edge[n1][n2]['perturb_radius'] = upper
-            total_rounding_error = round_up_error
+            total_rounding_error += round_up_error
```

Comment:
Why += instead of = ? I think the suggested addition is not necessary, because round_up_error from line 182 already contains total_rounding_error as a summand.